resources/js/components/utils/classnames.ts

```diff
@@ -0,0 +1,3 @@
+import classnames from "classnames";
+
+export const cx = classnames;
```

Comment:
`export { default as cx } from "classnames";` forma skrócona, jak chcesz to użyj, nie będę się czepiać

Reply:
Jakoś nie mogłem zgadnąć składni tego reëxportu i dlatego tak dałem, poprawię.

Tylko bez spacji :)

Reply:
Zrobione.